internal/summoner/acquire/headlessNG.go

```diff
@@ -231,7 +239,7 @@ func PageRender(v1 *viper.Viper, mc *minio.Client, logger *log.Logger, timeout t
 			});
 		}
 
-		function retry(fn, retriesLeft = 3, interval = 1000) {
+		function retry(fn, retriesLeft = 3, interval = 3000) {
```

Comment:
If I remember right, that interval was pretty arbitrary. Would it make sense to make it a multiple of HeadlessWait?